plugins/generator-1.16.4/forge-1.16.4/biome.definition.yaml

```diff
@@ -0,0 +1,11 @@
+templates:
+  - template: json/biome.json.ftl
+    writer: json
+    name: "@MODDATAROOT/worldgen/biome/@registryname.json"
+  - template: biome_builder.json.ftl

```

Comment:
You probably need `json/` here

Reply:
Oops, sorry. I copied and pasted the code of my data pack generator.